onnxruntime/test/ir/utils_test.cc

```diff
@@ -207,7 +207,7 @@ static void UpdateSubgraphWhenRemovingNode(bool include_nested = false) {
   auto& node_to_remove = *graph.GetNode(1);
   const auto& if_node = *graph.GetNode(2);
```

Comment:
We might want to open a PR to test some of the newly added graph_utils.

Reply:
Will check code coverage. If they're all covered by existing optimizers it seems low priority (to me) to add direct tests of the functions in graph_utils.